support/color/print.go

```diff
@@ -0,0 +1,93 @@
+package color
+
+import "github.com/pterm/pterm"
+
+/*
+Package color provides utilities for rendering and printing colorized text using color tags.
+
+The color tags are automatically parsed and rendered.
+
+Supported Features:
+1. Style Tags:
+   Use predefined style tags to format text. Example:
+   color.Print("<suc>he</><comment>llo</>, <cyan>wel</><red>come</>")
```

Comment:
Is there a rule that describes the tags, please?

Reply:
Don't we have `color.Red().Println` and other methods?

Reply:
> Is there a rule that describes the tags, please?

https://gookit.github.io/color/#/?id=html-like-tag-usage

Reply:
> Don't we have `color.Red().Println` and other methods?

It’s not just about simple colors like red or green. We also need to style text with bold, italic, underline, and use a wider range of RGB colors. For this, using gookit/color with its flexible color tag configuration is highly recommended.